src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -196,42 +197,42 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
         } 
     });
 
+    @computed get overlapStrategySelector() {
+        if (this.store._selectionInfo.isComplete) {
```

Comment:
should probably return null if not complete.  i'm surprised this passes ts.   shouldn't this be your MobxPromiseView?

Reply:
oh you're right